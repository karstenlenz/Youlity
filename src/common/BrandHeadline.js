import React from 'react'
import styled from 'styled-components/macro'
import PropTypes from 'prop-types'

BrandHeadline.propTypes = {
  firstWord: PropTypes.string.isRequired,
  secondWord: PropTypes.string.isRequired,
}

export default function BrandHeadline({ firstWord, secondWord }) {
  return (
    <Wrapper>
      <SpanStyled color="green">{firstWord}&nbsp; </SpanStyled>{' '}
      <SpanStyled>{secondWord}</SpanStyled>
    </Wrapper>
  )
}
const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  margin: 0 auto 15px;
`

const SpanStyled = styled.span`
  display: inline;
  font-size: 1.331em;
  color: ${(props) => {
    return props.color === 'green' ? 'var(--secondary)' : 'var(--primary)'
  }};
  margin: 0;
`
