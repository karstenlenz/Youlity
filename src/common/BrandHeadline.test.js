import { render } from '@testing-library/react'
import React from 'react'
import BrandHeadline from './BrandHeadline'

describe('The BrandHeadline component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <BrandHeadline firstWord="first" secondWord="second" />
    )
    expect(container).toMatchSnapshot()
  })
})
