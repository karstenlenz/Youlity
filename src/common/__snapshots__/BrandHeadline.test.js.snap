// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The BrandHeadline component renders correctly 1`] = `
<div>
  <div
    class="BrandHeadline__Wrapper-sc-1gsl6zz-0 kOqNrf"
  >
    <span
      class="BrandHeadline__SpanStyled-sc-1gsl6zz-1 eGCGLG"
      color="green"
    >
      first
      Â  
    </span>
     
    <span
      class="BrandHeadline__SpanStyled-sc-1gsl6zz-1 idxAxg"
    >
      second
    </span>
  </div>
</div>
`;
